internal/plugintest/util.go

```diff
@@ -68,3 +70,65 @@ func symlinkDirectoriesOnly(srcDir string, destDir string) error {
 
 	return nil
 }
+
+// CopyFile copies a single file from src to dest.
+func CopyFile(src, dest string) error {
+	var srcFileInfo os.FileInfo
+
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return err
```

Comment:
Nit: It might be good to wrap these errors (here and below) with what we were trying to do so folks can troubleshoot any issues easier:

```go
		return fmt.Errorf("unable to open file: %w", err)
```